test_enclave/src/pcl_key.bin

```diff
@@ -0,0 +1 @@
+""33DDUUffww����
```

Comment:
We should update `.gitattributes` so this file is treated as binary (suggest copying from mobilecoin.git)

Reply:
added